src/osd/OSDMap.h

```diff
@@ -714,6 +714,7 @@ class OSDMap {
   void get_all_osds(set<int32_t>& ls) const;
   void get_up_osds(set<int32_t>& ls) const;
   void get_out_osds(set<int32_t>& ls) const;
+  void get_out_existing_osds(std::set<int32_t>& ls) const;
```

Comment:
I'm not sure the resolution is correct here, we should not be using `std::` to be consistent.

Reply:
But it's a header so *all* references to std:: objects should be fully qualified. There is no "using namespace std" in the file so the lines above it are incorrect and only work by good fortune :) If anything, all the other lines need to be fully qualified but I don't believe that is in the scope of the backport and would tend to obfuscate git history anyway so probably not worth it at this time.